gitPractice/1-gitHomework.js

```diff
@@ -0,0 +1,45 @@
+//? practice to undrestand difference between const and let.
+//const and let are variable, on the other hand const is better than let because const variable won't be reasigned however let could be reasigned.
+// In conclusion:
+// "Var" variable can be updated and re-declared.don't need initialized.
+// "let" variable can be updated but not re-declared. don't need initialized.
+// "const" variable neither be update and re-declared. needs initialized.
+
+//?practice git and read the codeyourfuture git syllabus.
+//I read the git from codeYourFuture syllabus and also I watched the https://www.youtube.com/watch?v=BCQHnlnPusY and https://www.youtube.com/watch?v=iv8rSLsi1xo
+// Now I know how I can make a repo and how I can commit and add more commit, eventhough we could do the same in VS, git and gitHub.
+// In conclusion I need more practice to pull and push and make changes on it to clear all the options.
+
+//? git push, git pull, what is stage area in git, what is remote repository in git
+//? What is differece between local repository and remote repo in github.
+//"gitPull": If we want to update our branchs with our team mate, we need to write gitpull in terminal to check what have changed, on the other hand we could use git adn github.
+//"StageArea": is a temporary place to put our files to the up. in source control We need to click on + for each files we feel it could be finished, we could press the +.
+//"gitPush": If we want to send our changes in branch, we can write gitpush in terminal and also we could use git or github to update our branches.
+//"remoteRepository" I saw https://www.youtube.com/watch?v=lR_hYwCAaH4&list=PLRqwX-V7Uu6ZF9C0YMKuns9sLDzK6zoiV&index=10
+
+
+
+//? git cheatsheet -> find it.
+//Git CLI cheatsheet. This is a terminal which we could write rather than use git. However we could see the changes on the git as well.
+//"Terminal":
+//Using the terminal navigate to a specified folder (using cd)
+//Move up a level in a folder directory (using cd ..)
+//Cancel a running task in the terminal (using ctrl+c)
+// I could not find Windows WSL. I know that it is for linux.
+
+//?different between git and github
+//This is the difference between Git and GitHub. Git is a way of storing and sharing files, and GitHub is one website where you can use Git.
+
+
+
+//? search about stack overflow debogger platform
+// I could find any problem or any question about everything in coding and solve my problems.
+// The most important overflow debugger is MDN.
+
+//? codebar
+//I could ask my questions and aloso joint to the for any workshop what I am interested. 
+
+//? fix linkD profile.
+// I wrote more about myself in linkdin and explaned what I did for my clients and office.
+
```

Comment:
Great @RominaBideh  :)

Reply:
@ellietms If is it possible. Could you please help me with my CV as well?
Thank you in advance

Reply:
@RominaBideh  at first finish these, we can do your CV later when you are comfortable with git